src/vs/workbench/contrib/positronPlots/browser/positronPlots.contribution.ts

```diff
@@ -100,6 +100,17 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 		}
 	});
 
+Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
+	.registerConfiguration({
+		properties: {
+			'positron.plots.freezeSlowPlots': {
+				type: 'boolean',
+				default: true,
+				description: localize('positron.plots.freezeSlowPlotsSetting', "Freeze slow plots to prevent them from being re-rendered when the viewport size changes. This can improve performance for large plots."),
```

Comment:
_Super_ minor nit, but I think this is avoiding the computation vs improving performance.

> "Freeze slow to generate plots at a fixed size to avoid re-rendering on viewport changes, improving responsiveness of the IDE when working with complex charts."